core-services/image-mirroring/knative/mapping_knative_v0_19_quay

```diff
@@ -1,20 +1,20 @@
-registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-sender quay.io/openshift-knative/knative-eventing-test-wathola-sender:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-event-sender quay.io/openshift-knative/event-sender:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-event-sender quay.io/openshift-knative/event-sender:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-heartbeats quay.io/openshift-knative/heartbeats:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-heartbeats quay.io/openshift-knative/heartbeats:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-performance quay.io/openshift-knative/performance:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-performance quay.io/openshift-knative/performance:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-print quay.io/openshift-knative/print:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-print quay.io/openshift-knative/print:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-recordevents quay.io/openshift-knative/recordevents:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-recordevents quay.io/openshift-knative/recordevents:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-request-sender quay.io/openshift-knative/request-sender:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-request-sender quay.io/openshift-knative/request-sender:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v0.19.2
+registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v0.19
```

Comment:
I see we want two tags, how this helps ? :thinking:

Reply:
the 0.19.x  is used on dependenign repos for tests 

that's not the change ....

the change is... I remove the `knative-eventing-test-` prefix ...

Reply:
I see that. Wanted to check about multiple tags, looks like this to prevent frequent updates on dependent repos using those images@v0.19.